### PR TITLE
[No JIRA] Update .nvmrc to latest node 7

### PR DESCRIPTION
**Pull request by k0nserv**
+ [x] For any new components I've made sure that the style can be extended by the consumer using a `className` override.
+ [x] For any new files I've included the [Apache 2.0 License header](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/formatters/license-header.js#L2-L16) at the top of the file.
+ [x] I've updated `changelog.md` for any changes that need to be highlighted in the changelog.
+ [x] If I've updated `npm-shrinkwrap.json` those changes are intentional.

### CODE REVIEW

**Comment by k0nserv** [comment]
Yeah a script seems unecessary. Update those now

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - 7.5.0
+  - 7.10.1
```
Comment (georgegillams) [inline .travis.yml]: Why do we even still have a travis.yml file?

Comment (georgegillams) [inline .travis.yml]: Haha I was thinking about drone... ignore me

Comment (k0nserv) [inline .travis.yml]: You're thinking  of `.drone.yml`

**Comment by georgegillams** [comment]
LGTM - all building and running fine for me :)

**Pull request merged**